addons/l10n_ar_withholding/models/account_tax.py

```diff
@@ -6,14 +6,73 @@ class AccountTax(models.Model):
 
     _inherit = 'account.tax'
 
+    l10n_ar_type_tax_use = fields.Selection(
+        selection=[
+            ('sale', 'Sales'),
+            ('purchase', 'Purchases'),
+            ('none', 'Other'),
+            ('supplier', 'Vendor Payment Withholding'),
+            ('customer', 'Customer Payment Withholding')
+        ],
+        compute='_compute_l10n_ar_type_tax_use', inverse='_inverse_l10n_ar_type_tax_use',
+        string="AFIP Tax Type"
+    )
     l10n_ar_withholding_payment_type = fields.Selection(
-        [('supplier', 'Supplier'), ('customer', 'Customer')], 'Argentinean Withholding type',
-        compute="_compute_l10n_ar_withholding_payment_type", store=True, readonly=False)
-
+        selection=[('supplier', 'Vendor Payment'), ('customer', 'Customer Payment')],
+        string="AFIP Withholding Type",
+        help="Withholding tax for supplier or customer payments.")
+    l10n_ar_tax_type = fields.Selection(
+        string='WTH Tax Type',
+        selection=[
+            ('earnings', 'Earnings'),
+            ('earnings_scale', 'Earnings Scale'),
+            ('iibb_untaxed', 'IIBB Untaxed'),
+            ('iibb_total', 'IIBB Total Amount'),
+        ]
+    )
     l10n_ar_withholding_sequence_id = fields.Many2one(
-        'ir.sequence', 'Withholding Number Sequence', copy=False, check_company=True,
+        'ir.sequence',
+        string='AFIP Withholding Sequence',
+        copy=False, check_company=True,
         help='If no sequence provided then it will be required for you to enter withholding number when registering one.')
+    l10n_ar_code = fields.Char('AFIP Code', help="Earning withholding regimen.")
+    l10n_ar_non_taxable_amount = fields.Float(
+        string='WTH Non Taxable Amount',
+        digits='Account',
+        help="Until this base amount, the tax is not applied."
+    )
+    l10n_ar_withholding_minimum_threshold = fields.Float(
+        string="WTH Minimum Treshold",
+        help="If the calculated withholding tax amount is lower than minimum withholding threshold then it is 0.0.")
+    l10n_ar_state_id = fields.Many2one(
+        'res.country.state', string="AFIP State", ondelete='restrict', domain="[('country_id', '=?', country_id)]")
+    l10n_ar_scale_id = fields.Many2one(
+        comodel_name='l10n_ar.earnings.scale',
```

Comment:
@pablohmontenegro we are missing the label here.
Can you add it `string="AFIP Scale"
Thanks!